doc/classes/InputEventKey.xml

```diff
@@ -24,6 +24,12 @@
 				Returns a [String] representation of the event's [member keycode] and modifiers.
 			</description>
 		</method>
+		<method name="as_text_location" qualifiers="const">
+			<return type="String" />
+			<description>
+				Returns a [String] representation of the event's [member location].  This will be a blank string if the event is not specific to a location.
```

Comment:
```suggestion
				Returns a [String] representation of the event's [member location]. This will be a blank string if the event is not specific to a location.
```